README.md

```diff
@@ -174,7 +174,7 @@ func TestSomething(t *testing.T) {
 // This time using a placeholder. Placeholders might be used when the
 // data being passed in is normally dynamically generated and cannot be
 // predicted beforehand (eg. containing hashes that are time sensitive)
-func TestSomethingElse(t *testing.T) {
+func TestDoSomethingWithPlaceholder(t *testing.T) {
```

Comment:
This makes sense. Not sure why we have the `Do` part but I can see how having `TestSomethingWithPlaceholder` makes things a little clearer.

Importantly, the docstrings must start with the (new) function names.

Reply:
Thanks for giving a positive opinion :)
Surely, I forget to revise the docstrings... [I revised it](9e16738c413647272784610361e4694bbaa27609) :)
Whether to have the `Do` part, I'll follow the previous example.